### DISCUSSION TITLE
Can I change border style e.g. dotted lines or *** etc ?

### DISCUSSION THREAD

**Discussion by N1ck9141** [i3/i3#5783] [Q&A]
Sorry if this is a foolish question, but in my i3 config I use the standard. 

`for_window [class="^.*"] border pixel 2 `

this removes window decoration and creates a simple 2 pixel border around all windows as you know, I have the active window have a different color to make it easy to identify.

I was wondering if there was a way to change the style of this basic pixel line? So i could make it look dotted? 

OR is this something that Picom might be able to do ? 

Thank you

**Comment by orestisfl** [comment]
There is no i3 option to do that
